packages/core/src/lib/modal.svelte

```diff
@@ -60,19 +70,21 @@ onMount(() => {
 });
 </script>
 
+<svelte:window on:keydown={handleEscapePress} />
+
 <div
   class={cx(
     'fixed left-0 top-0 z-50 flex h-full w-full items-center justify-center bg-black bg-opacity-40',
     {
       invisible: !open,
```

Comment:
I know this is existing code, but IMO a closed modal should simply not be rendered. Can this `invisible` be swapped with an `#if`?

```svelte
{#if open}
  <svelte:window on:keydown={...} />
  <!-- ... -->
{/if}
```

As I'm writing this out, I bet there's a decent change the class="invisible" (as opposed to "do not render" is why `use:clickOutside` isn't working